compiler/src/org.graalvm.compiler.asm.aarch64/src/org/graalvm/compiler/asm/aarch64/AArch64MacroAssembler.java

```diff
@@ -919,17 +919,25 @@ public void cset(int size, Register dst, ConditionFlag condition) {
         super.csinc(size, dst, zr, zr, condition.negate());
     }
 
+    private static ExtendType getLSLExtendType(int size) {
```

Comment:
Probably drop the LSL from the method name. Graal's ExtendType enum doesn't mention LSL.

Reply:
I wanted to have LSL in the name, since that is what I'm trying to encode there (it just so happens which type is LSL depends on the instruction size)